src/models/model.cpp

```diff
@@ -105,6 +99,7 @@ model::model(const model& other) :
   for (const auto& ptr : other.m_layers) {
     if (ptr == nullptr) { LBANN_ERROR("unexpected null pointer"); }
     auto* old_layer = ptr.get();
+    /// @todo BVE FIXME - not sure copy does what I think that it should
```

Comment:
This is concerning.

Reply:
We discussed this.  I have removed the comment.